plugins/etc/httpd/conf.d/pulp_rpm.conf

```diff
@@ -16,19 +16,23 @@
 AddType application/x-pkcs7-crl .crl
 AddType application/x-x509-ca-cert .crt
 
-# -- HTTPS Repositories ---------
-Alias /pulp/repos /var/www/pub/yum/https/repos
+# -- Yum Repositories ---------
+<Location /pulp/repos/>
```

Comment:
Consider whether it's worth making sure that our code does not generate links to /pulp/repos/ anymore (but to /pulp/content instead) so this is only necessary for legacy yum configs.